### PR TITLE
Feat: input

### PR DESCRIPTION
**Pull request by vivekmehto**
https://user-images.githubusercontent.com/76930847/152172639-471e64b9-1688-4641-8009-376e40b5752b.mov

### CODE REVIEW

File: components/input/input.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  font-size: 1.2rem;
+  font-weight: 600;
+}
+
+.input-wrapper label input {
+  border: 1px solid var(--grey);
+  padding: 0.5rem 0.8rem;
+  margin: 0.8rem 0;
+  border-radius: 2px;
+}
```
Comment (ttahm3d) [inline components/input/input.css:17]: label could have been given a generic class `.input-label` and sane case for `input` also

**Review by ttahm3d — COMMENTED** [review]
Only once change try to give classes to label and input items

File: components/input/input.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+.input-error label input {
+  border: 1px var(--red) solid;
+}
+
+input:hover {
+  box-shadow: 2px 2px 2px var(--grey);
+}
+
+/* Radio and Checkbox */
```
Comment (ImrozNezami) [inline components/input/input.css:37]: Remove comment

Comment (vivekmehto) [reply components/input/input.css:37]: But why ??

File: components/input/input.css (hunk 1 above)
Comment (vivekmehto) [reply components/input/input.css:17]: Okay working on it. Thank you

**Review by ttahm3d — APPROVED** [review]

**Pull request merged**